user-guide/modules/ROOT/pages/boost-history.adoc

```diff
@@ -43,11 +44,19 @@ While many Boost libraries have been incorporated into the C++ Standard Library,
 
 * boost:json[] : A JSON library for encoding, decoding, and manipulating JSON data. Added to Boost in *January 2021*.
 
-These libraries, among others, showcase the value and versatility of Boost in providing advanced functionality beyond the scope of the pass:[C++] standard library.
+These libraries, among many others, showcase the value and versatility of Boost in providing advanced functionality beyond the scope of the pass:[C++] Standard Library. As well as the xref:common-introduction.adoc[Common] and xref:advanced-introduction.adoc[Advanced] scenarios highlighted in this documentation, Boost libraries are used in highly specialized niche markets, ranging from probability theory to astronomy to mass spectroscopy. Open source isn't just for nerds and researchers. Real world programming challenges, irrespective of whether they are open or closed source, can benefit enormously from the thought and experience that has gone into the libraries.
```

Comment:
> Boost libraries are used in highly specialized niche markets

I would use "highly specialized" or "niche", but not both. Consider using "applications" instead of "markets"